### PR TITLE
wolfssl: add wolfssl patches

### PR DESCRIPTION
**Pull request by expressvpn-raihaan-m**
## Description
Fixes bugs with wolfSSL 5.6.4

Patches included:
* 0001 - patch for pqc crash fix
* 0002 - patch for arm64 assembly issues

## Motivation and Context
Prevents our workarounds around these issues

## How Has This Been Tested?
iOS was tested manually

@xv-ian-c has the PQC patch running on one of his systems

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All active GitHub checks are passing  
- [ ] The correct base branch is being used, if not `main`

### CODE REVIEW

**Review by expressvpn-tom-l — APPROVED** [review]
LGTM

File: ios/autotools-ios-helper.sh [hunk 1]
```diff
@@ -24,7 +24,7 @@ OQS_BUILD=${OQS_BUILD:-"$(pwd)/../liboqs/build_universal"}
 
 build() {
     # Compiler options
-    export OPT_FLAGS="-O3 -fno-inline"
+    export OPT_FLAGS="-O3"
```
Comment (xv-ian-c) [inline ios/autotools-ios-helper.sh:27]: Might be worth mentioning in the commit message that this because unnecessary due to the second patch?

**Review by xv-ian-c — APPROVED** [review]
LGTM with a minor commit message nit.

I'm curious if the noinline stuff is really needed with the `"cc"` fix in place, but that's an upstream question.

**Comment by expressvpn-raihaan-m** [comment]
> LGTM with a minor commit message nit.
> 
> I'm curious if the noinline stuff is really needed with the `"cc"` fix in place, but that's an upstream question.

Honestly, I don't believe so but they seem to be doing it across their codebase so I think they intend to reduce inlining around this code.

**Pull request merged**